### PR TITLE
yank: update to 1.1.0.

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

**Comment by maxice8** [comment]
It failed last time it was updated so i'm asking for review

**Comment by prspkt** [comment]
@maxice8 sure.

File: srcpkgs/yank/template [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 license="MIT"
+homepage="https://github.com/mptre/yank"
+changelog="https://raw.githubusercontent.com/mptre/yank/master/CHANGELOG.md"
 distfiles="https://github.com/mptre/yank/releases/download/v${version}/yank-${version}.tar.gz"
-checksum=20ba1b01a7514f2f670702e83846f42429e2e90491dc1f679c8319a45a0382de
+checksum=f197107cd604c41ef3c2390175bd34e5ad3c7cf1feafedefe357ba75149f1c59
+
+pre_build() {
+	sed -i 's|^CFLAGS *+=|override CFLAGS +=|g' Makefile
+}
```
Comment (asergi) [inline srcpkgs/yank/template]: You can get rid of `pre_build()` simply by using `make_use_env=yes`.

**Comment by asergi** [comment]
Merged manually in ab0278ded3f017ef12f2e5bec9c2a7d8cc56734d. Thanks.

**Pull request closed without merging**